src/client/ManagedNetwork.js

```diff
@@ -111,11 +107,11 @@ export default class MangedNetwork {
     }
 
     /**
-     * @param {NetworkName} networkName
+     * @param {string} networkName
      * @returns {this}
      */
     setNetworkName(networkName) {
-        this._ledgerId = _ledgerIdToLedgerId(networkName);
+        this._ledgerId = NetworkName.networkIdFromName(networkName).toString();
```

Comment:
Change `this._ledgerId` to be of type `NetworkName` instead and stringify it only when needed.